src/puppetlabs/puppetdb/http/nodes.clj

```diff
@@ -27,34 +27,27 @@
       (http/status-not-found-response "node" node))))
 
 (defn routes
-  [version]
+  [{:keys [scf-read-db url-prefix api-version] :as globals}]
   (app
    []
-   (http-q/query-route :nodes version http-q/restrict-query-to-active-nodes')
+   (http-q/query-route :nodes globals http-q/restrict-query-to-active-nodes')
 
    [node]
-   {:get
-    (-> (fn [{:keys [globals]}]
-          (node-status version
-                       node
-                       (:scf-read-db globals)
-                       (:url-prefix globals)))
-        ;; Being a singular item, querying and pagination don't really make
-        ;; sense here
-        (validate-query-params {}))}
+   {:get (-> (node-status api-version node scf-read-db url-prefix)
+             constantly
```

Comment:
very minor point of preference, but I find the threading through constantly to require a bit of a double take compared to wrapping the node-status call at the top level. There's a couple spots where we're doing it in here -- take it or leave it.